USERGUIDE.md

```diff
@@ -59,83 +52,44 @@ If you don't want to use project settings, you have to pass this information one
 Config.userDefaults?.set("MY_CUSTOM_ID", forKey: Config.Key.subscriptionKey.rawValue)
 ```
 
-<div id="usage"></div>
+<a id="usage"></a>
 
-## Usage
+### Usage
 
-* [Overriding configuration on runtime](#runtime-conf)
+* [Create a MiniApp](#create-mini-app)
+* [Mini App Features](#mini-app-features)
+    * [Retrieving Unique ID](#retrieve-unique-id)
+    * [Requesting Location Permissions](#request-location-permission)
+    * [Custom Permissions](#custom-permissions)
+    * [Share Mini app content](#share-mini-app-content)
+    * [Retrieve User Profile details](#retrieve-user-profile-details)
 * [Load the Mini App list](#load-miniapp-list)
 * [Get a MiniAppInfo](#get-mini-appinfo)
-* [Create a MiniApp](#create-mini-app)
-* [Communicate with MiniApp](#MiniAppMessageDelegate)
-* [Customize history navigation](#navigation)
-* [Custom Permissions](#custom-permissions)
 * [List Downloaded Mini apps](#list-downloaded-mini-apps)
-* [Retrieve User Profile details](#user-profile-details)
-* [Orientation Lock](#orientation-lock)
-* [Access Token info](#access-token-info)
-
-<div id="runtime-conf"></div>
-
-### Overriding configuration on runtime
----
-Every call to the API can be done with default parameters retrieved from the project .plist configuration file, or by providing a `MiniAppSdkConfig` object during the call. Here is a simple example class we use to create the configuration in samples below:
-
-```swift
-class Config: NSObject {
-    class func getCurrent() -> MiniAppSdkConfig {
-        return MiniAppSdkConfig(baseUrl: "https://your.custom.url"
-                                rasAppId: "your_RAS_App_id",
-                                subscriptionKey: "your_subscription_key",
-                                hostAppVersion: "your_custom_version",
-                                isTestMode: true")
-    }
-}
-```
-*NOTE:* `RMAHostAppUserAgentInfo` cannot be configured at run time.
+* [Advanced Features](#advanced-features)
+    * [Overriding configuration on runtime](#runtime-conf)
+    * [Customize history navigation](#custom-navigation)
+    * [Opening external links](#Opening-external-links)
+    * [Orientation Lock](#orientation-lock)
 
+<a id="create-mini-app"></a>
 
-<div id="load-miniapp-list"></div>
-
-### Load the `MiniApp` list:
+### Create a MiniApp for the given `MiniAppId` :
 ---
-MiniApp library calls are done via the `MiniApp.shared()` singleton with or without a `MiniAppSdkConfig` instance (you can get the current one with `Config.getCurrent()`). If you don't provide a config instance, values in custom iOS target properties will be used by default. 
-
-```swift
-MiniApp.shared().list { (result) in
-	...
-}
-```
+**API Docs:** [MiniApp.create](https://rakutentech.github.io/ios-miniapp/Classes/MiniApp.html), [MiniAppDisplayDelegate](https://rakutentech.github.io/ios-miniapp/Protocols/MiniAppDisplayDelegate.html)
```

Comment:
In the case of iOS, you don't need to manually add the URL: Jazzy can generate a link automatically if you put a class name or method reference in backticks. For example, this should work:
```suggestion
**API Docs:** `MiniApp.create(appId:completionHandler:messageInterface:)`, `MiniAppDisplayDelegate`
```

See the Jazzy docs:
https://github.com/realm/jazzy#supported-documentation-keywords